externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/kyuubi/SparkConsoleProgressBar.scala

```diff
@@ -72,6 +73,22 @@ class SparkConsoleProgressBar(
     }
   }
 
+  /**
+   * Use stageId to find stage's jobId
+   * @param stageId stageId
+   * @return jobId,if not exists, return -1
+   */
+  private def findJobId(stageId: Int): Int = {
```

Comment:
return `Option[Int]` then use `isEmpty/isDefined` to determine whether it is empty